samples/drivers/led_ws2812/README.rst

```diff
@@ -102,6 +102,8 @@ This sample uses different drivers depending on the selected board:
 I2S driver:
 
 - thingy52_nrf52832
+- nrf5340dk_nrf5340 (1.8V logic level, make sure to use a logic level shifter!)
+    - works with nrf7002dk_nrf5340 too
```

Comment:
Please remove as this is an out-of-tree board, or reword along the lines of "can be used [or "should work"] for other boards featuring an nRF5340 host processor" if that is the case.


```suggestion
```

Reply:
Updated the README as you suggested.